pkg/url/kubernetes.go

```diff
@@ -0,0 +1,108 @@
+package url
+
+import (
+	"fmt"
+	"sort"
+
+	routev1 "github.com/openshift/api/route/v1"
+	componentlabels "github.com/openshift/odo/pkg/component/labels"
+	"github.com/openshift/odo/pkg/localConfigProvider"
+	"github.com/openshift/odo/pkg/occlient"
+	"github.com/pkg/errors"
+	"k8s.io/klog"
+)
+
+type kubernetesClient struct {
+	generic
```

Comment:
What is the significance of this? :thinking:

Reply:
Added comments